### ISSUE TITLE
Great initiative!

### ISSUE THREAD

**Issue by Jon-guy30** [Darthagnon/idontcareaboutcookies-backup#1]
Just to get the conversation started here how the heck did you get the source code? I've looked and it's nowhere to be found. I was surprised since I was under the impression it's open source. Now it's proprietary?! I'm no programmer so I can't help you with this but I can spread the word where I can.

**Comment by Darthagnon** [comment]
Spoiler: it's not the source code. Thank you for the kind words, though. 

This repo contains the packaged extensions, last known good (pre-Avast acquisition) version. However, said extensions are effectively ZIP files and can be unzipped and all the source code is there (web languages, HTML/JS/CSS are always visible).

I'm not a good programmer, but I know enough tinkering to download CRX files from web stores. 

- For Chromium: everything is on the Chrome web store, use a bookmarklet as specified in the Readme
- For Firefox: visit the Mozilla Extension Store from a non-firefox browser, and you get a "download file" link. 
- For Pale Moon: a list of previous versions of extensions are saved in their extension repository. Spoof your user agent as Pale Moon, and you can download.
- For Opera, you can just download the extension by clicking a button, but you gotta be fast, as new versions are added and old versions are then unavailable.

**Comment by Darthagnon** [comment]
@Jon-guy30 Someone else has deconstructed the extension and done a hard fork from the source code (allowing future development) here: https://github.com/OhMyGuus/I-Dont-Care-About-Cookies

**Comment by Jon-guy30** [comment]
Ah, cool. These replacement versions ought to be signed too if they aren't and uploaded to the Mozilla addon repository. I've had trouble installing extensions via GitHub and I kind of don't want to disable extension signing in the browser. I'm going to live without this addon for a while and see how the situation develops.

**Comment by Darthagnon** [comment]
Try using the filterlist in uBlock Origin - the original should still be fine. It's linked in the Readme.

**Comment by Darthagnon** [comment]
I can't sign this backup version and upload it to any webstores, because (a) I don't know how, (b) I don't intend to become the new maintainer or develop it further, only to provide a backup; (c) I don't know what license the original is distributed under, nor do I wish to compete with the original author (having interacted with him in the past, he seemed a decent guy; given the current economy, I can't really blame him for selling out). Also (d) I always disable extension signing in my browsers, it's just a form of DRM. 

Maybe OhMyGuus, since he's developing it further, might do so.

**Comment by Jon-guy30** [comment]
Yep. I'm already using `ublock-annoyances`, it doesn't work on Google though, IDCAC did not either. I've corresponded with the author in the past too, very responsive. For a while he got rid of the Google cookie page but it reappeared. :( I didn't bother to bug him about it again either.
